contrib/fwupd.spec.in

```diff
@@ -218,6 +218,8 @@ Summary: fwupd wrapper for Qubes OS - dom0 scripts
 Requires:   gcab
 Requires:   fwupd >= 1.5.7
 Requires:   libjcat >= 0.1.6
+Requires:   python3-packaging
+Requires:   sequoia-sqv
```

Comment:
this needs to be surrounded by the qubes `if` section -- we can't add those for all systems.

Reply:
Ohh it is! ignore me.